template_project/cmake/LookupClang.cmake

```diff
@@ -0,0 +1,50 @@
+set(CLANG_VERSIONS 15 14)
+
+set(LLVM_DIR "" CACHE PATH "Lookup path for the llvm installation.")
+# https://stackoverflow.com/questions/28613394/check-cmake-cache-variable-in-toolchain-file
+# Problem: CMake runs toolchain files multiple times, but can't read cache variables on some runs.
+# Workaround: On first run (in which cache variables are always accessible), set an intermediary environment variable.
```

Comment:
A workaround in the cmake tool chain related cmake file.

Reply:
I will look into this.